From: Xiu Jianfeng <xiujianfeng@huawei.com>
Subject: writeback: cleanup bdi_sched_wait()
Date: Wed, 13 Jul 2022 20:53:14 +0800

bdi_sched_wait() is no longer used since commit 839a8e8660b6 ("writeback:
replace custom worker pool implementation with unbound workqueue"), so
remove it.

Link: https://lkml.kernel.org/r/20220713125314.171345-1-xiujianfeng@huawei.com
Signed-off-by: Xiu Jianfeng <xiujianfeng@huawei.com>
Reviewed-by: Jan Kara <jack@suse.cz>
Reviewed-by: Johannes Thumshirn <johannes.thumshirn@wdc.com>
Acked-by: Jens Axboe <axboe@kernel.dk>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 include/linux/backing-dev.h |    6 ------
 1 file changed, 6 deletions(-)

--- a/include/linux/backing-dev.h~writeback-cleanup-bdi_sched_wait
+++ a/include/linux/backing-dev.h
@@ -140,12 +140,6 @@ static inline bool mapping_can_writeback
 	return inode_to_bdi(mapping->host)->capabilities & BDI_CAP_WRITEBACK;
 }
 
-static inline int bdi_sched_wait(void *word)
-{
-	schedule();
-	return 0;
-}
-
 #ifdef CONFIG_CGROUP_WRITEBACK
 
 struct bdi_writeback *wb_get_lookup(struct backing_dev_info *bdi,
_
